\documentclass[11pt]{report}
\title{\textbf{Genotypic Evaluation of Carbapenemases Producing Strains Isolated from Different Biological Materials}}
\author{Simona Debilio}
\date{23 March 2017}

\usepackage{graphicx}
\usepackage[utf8x]{inputenc}
\usepackage{grffile}
\usepackage[margin=1in]{geometry}
\usepackage[english]{babel}
\setlength{\parindent}{0pt}

\begin{document}

\maketitle

\tableofcontents

\chapter{Introduction}
One of the most important discoveries in human history has been the identification of substances that were able to fight and defeat bacterial infections.
These substances, called “antibiotics”, had an extraordinary impact on the outcome of bacterial infections and, consequently, helped to extend life expectancy \cite{ventola2015antibiotic}.

The first antibiotic was penicillin, discovered by Sir Alexander Fleming in 1928 (Figure 1.1) and following developed by chemical companies.

\begin{figure}[htp]
\centering
\includegraphics[scale=1.10]{immagini gamalero/Fleming_piastra.jpg}
\caption{"It has been demonstrated that a species of penicillium produces in culture a very powerful antibacterial substance which affects different bacteria in different degrees. Generally speaking it may be said that the least sensitive bacteria are the Gram-negative bacilli, and the most susceptible are the pyogenic cocci. [...] In addition to its possible use in the treatment of bacterial infections penicillin is centainly useful [...] for its power of inhibithing unwanted microbes on bacterial cultures so that penicillin insensitive bacteria can readily be isolated." Alexander Fleming }
\label{}
\end{figure}

\clearpage
At first, antibiotics were used to treat serious infections in the 1940s, and they had similar positive outcomes worldwide \cite{Spellberg2014}.

Unfortunately, after many decades since the use of the first antibiotics on patients, bacterial infections began to spread again and, more importantly, became a threat again.
This renewed spread of bacterial infections is due to the emergence of bacterial strains that are resistant to most of present antibiotics \cite{ventola2015antibiotic}.

The main causes that have provoked/caused this antibiotic resistance crisis, which is occurring worldwide, are antibiotic overuse and misuse.
In addition, there has been a lack in the development of new drugs by the pharmaceutical industry \cite{nature2013}.
Overall, this situation has a huge impact on human health (Figure 1.2).

\begin{figure}[htp]
\centering
\includegraphics[scale=0.80]{immagini gamalero/Antimicrobial_Spread2050.png}
\caption{In 2014 the UK Government commissioned the Review on Antimicrobial Resistance (AMR) to analyse the global problem of rising drug resistance. The process was completed in two years and some projections, based on the data gathered, showed that an increase in infection rates could mean 150 million people dying between now and 2050 \cite{review2014antimicrobial}}
\label{}
\end{figure}

As epidemiological studies have shown there is a direct relationship between the use of antibiotics and the emergence and spread of resistant bacteria strains \cite{huttner2013antimicrobial}.

Resistance can arise spontaneously through mutation (Figure 1.3), but it can also be acquired from genetic elements inherited from other bacteria.
The acquisition of mobile genetic elements (i.e. plasmids) is called “horizontal gene transfer”, and it allows not only the transfer among different members of the same species, but also among members of different species.

\clearpage
Furthermore, antibiotics cause the death of drug sensitive competitors, leaving only the resistant bacteria alive and able to reproduce themselves as the result of natural selection \cite{doi:10.1093/emph/eou024}.

\begin{figure}[htp]
\centering
\includegraphics[scale=0.57]{immagini gamalero/Bacteria_Resistance.png}
\caption{Most microbes can evolve rapidly thanks to their short reproductive cycle. During these frequent replications, mutations can arise; some of them can help the microbe surviving the exposure to antibiotics \cite{NHI_DrugResistance}.}
\label{}
\end{figure}

One way we can limit the spread of antibiotic resistant strains is by minimising the natural selection for resistance genes.
This can be achieved by reducing the use of antibiotics, avoiding their use unless it is strictly necessary, and enhancing infection prevention (e.g. isolating infected patients, and improving the hygiene).

Moreover, it is necessary to stop abusing antibiotics in agriculture \cite{Spellberg2014} \cite{doi:10.1093/emph/eou024} (Figure 1.4).
The excessive use of antibiotics in agriculture can lead to antibiotic resistance, due to the spread of molecules from the animal faeces to soil, water, and food. This phenomenon can cause the  involuntary ingestion of antibiotics in humans, and strengthening the natural selection for the microorganisms.

\begin{figure}[htp]
\centering
\includegraphics[scale=0.57]{immagini gamalero/Agricultural_Spread.jpg}
\caption{Antibiotics, when given to animals, kill most of the bacteria living in their intestines. Resistant strains can multiply and spread in the environment from their faeces \cite{CDC_AntibioticResistance}.}
\label{}
\end{figure}

\clearpage
Following the EU conference titled “The Microbial Threat”, which took place in 1998 in Copenhagen, antibiotic resistance became an official EU issue for the first time.
Since 2001, the European Council has highlighted the importance of reinforcing epidemiological surveillance, and improved the supervision of laboratories. The council also highlighted the need to create a coordinated structure at national level, in order to prevent and control the spread of antibiotic resistances.
In recent years Italy has seen the spread of Gram-negative bacteria, mostly belong to the \emph{Klebsiella pneumoniae} species that ascribed to the Enterobacteriaceae family, have become resistant to carbapenems (e.g. Imipenem and Meropenem).

A dramatically increasing trend in resistance has been observed between 2009 and 2011: while only the 1,3$\%$ of \emph{K. pneumoniae} strains isolated from blood showed a resistance in 2009, the percentage rose to 16$\%$ in 2010, and 26.7$\%$ in 2011 repectively (Figure 1.5 e 1.6).

\begin{figure}[htp]
\centering
\includegraphics[scale=0.60]{immagini gamalero/K.pneu_2009.png}
\caption{Klebsiella pneumoniae: proportion of invasive isolates resistant to carbapenems in 2009 \cite{ECDC_Surveillance}.}
\label{}
\end{figure}

\clearpage
\begin{figure}[htp]
\centering
\includegraphics[scale=0.60]{immagini gamalero/K.pneu_2015.png}
\caption{Klebsiella pneumoniae: proportion of invasive isolates resistant to carbapenems in 2015 \cite{ECDC_Surveillance}.}
\label{}
\end{figure}

These studies have confirmed the recent spread of multiresistant Enterobacteria in Italy, and they have shown why these bacteria represent a concrete threat for public health, as they are frequently the cause of infections, both in hospital and community environment \cite{circolare2013}.

Clinical microbiology laboratories play an essential role in monitoring the spread of carbapenemase-producing Enterobacteria (CPE).This role entails significant demands upon laboratory staff and resources. For example, medical personnel must become familiar with a series of technical procedures for rapid identification of resistant bacterial strains in order to apply appropriate measures to contain their spread; this is required to allow effective and rapid application of appropriate infection controls in response. The potential clinical impacts upon patients also requires microbiologists undergo constant methodological, technical and organizational training. A timely identification of the antibiotic-resistant strains allows an effective implementation of control over a particular infection.

\section{The definition of infectious diseases and transmission}
Infectious diseases are caused by pathogens, such as bacteria, viruses and fungi. When coming into contact with a host, pathogens are able to procreate and cause functional impairments. :  Hence,  diseases arise from these complex interactions between the host immune systems and the foreign organisms. Parasitism is the relationship between pathogens and hosts, where the pathogens take advantage of some vital functions of the host to replicate and survive. 

The human body is equipped with various defences against foreign substances and microorganisms. For example, skin and mucous are first line defence that possess antimicrobial properties which allow them to serve as effective barriers against bacterial invasion.

The process of invasion and proliferation of infectious microorganisms within the host body is called infection. Typically, there is an incubation period prior to onset of a disease. Incubation period is defined as the time elapsing from the exposure of an infectious agent and the first appearance of symptoms.  An incubation period can varies depending on the disease, and the establishment of the “infectious agent-host” relationships.

If symptoms appear during the infection, we have the onset of a “disease”, but the infection can also run without symptoms and, in that case, it is said to be an “asymptomatic infection”.
Contagious infectious diseases are caused by pathogens that are transmitted to receptive subjects.
Infectious disease which are not contagious require instead the intervention of suitable vectors and specific circumstances.
An effective prevention of infectious diseases can be achieved by removing one of their two causes: exposition to the pathogen and state of susceptibility \cite{EPICentro}.

\chapter{What is an antibiotic and how it works}
Antibiotics are secondary metabolites produced naturally by bacteria and other soil microorganisms.
In natural environment they probably play a defensive function for the organisms producing them.
From a pharmacological point of view, they have revolutionised medicine, providing a “universal” cure for infectious diseases that had remained untreatable for centuries.
Hundreds of molecules with antibacterial activity have been identified, and they have been divided into different classes on the base of the different chemical characteristics that distinguish the pharmacologically active molecule.
Although every antibiotic interferes at some level with the survival of the bacteria, their mechanism of action can be very different.
Antibiotics can alter the structure of the bacterial cell wall or cell membrane, with the energy metabolism, the synthesis of nucleic acids, or the protein synthesis (Figure 2.1).

Depending on the kind of antibiotic, there can be bactericidal effects, which directly cause the death of microorganisms, or bacteriostatic effects, which cause the inhibition of the bacterial reproduction, but not their death \cite{Leekha2011}.

Each antibiotic has its own spectrum of activity: some are effective on Gram-positive bacteria (e.g. penicillin), while others are effective on Gram-negative bacteria (e.g. cephalosporin).
Even though there are broad-spectrum antibiotics, which are able to remove many of the Gram-positive and Gram-negative bacteria, no antibiotic is active against all bacteria.
For this reason, it is essential to know the antibiotic range of action in order to obtain a targeted and effective pharmacological therapy.
Antibiotics distinguish themselves from other substances of microbial origin (e.g. toxins) because they can present high selectivity with regard to bacterial cells and low toxicity to eukaryotic cells.
This mechanism, which is called “selective toxicity”, allows them to eliminate the infection without damaging the patient.

\clearpage
\begin{figure}[htp]
\centering
\includegraphics[scale=0.700]{immagini gamalero/Antibiotic_Targets.jpg}
\caption{Antibiotics can have different modes of action: they can inhibit DNA, proteins, or cell wall  synthesis, but they can also interfere with the bacterial metabolism, or cause the impairment of nucleic acids.}
\label{}
\end{figure}

\section{$\beta$-lactam antibiotics}
$\beta$-lactam antibiotics are a class of broad-spectrum antibiotics which consists of all the antibiotics containing a $\beta$-lactam ring in their molecular structure, including penicillins, cephalosporins and carbapenems \cite{Pitout2005}.
These antibiotics present a nucleus (6-aminopenicillanic acid) connected to different lateral chains that affect their pharmacokinetic peculiarities and their range of action (Figure 2.2).

Penicillins have a bicyclic structure, and the $\beta$-lactam ring is the functional part of the molecule: if degraded the drug loses its effectiveness.
$\beta$-lactam antibiotics are active against many Gram-negative and Gram-positive bacteria.

\clearpage
\begin{figure}[htp]
\centering
\includegraphics[scale=0.35]{immagini gamalero/beta_lacatams.jpg}
\caption{Beta-lactam antibiotics are named after their beta-lactam ring: this is a four membered lactam with a nitrogen atom attached to the beta-carbon atom relative to the carbonyl.}
\label{}
\end{figure}

The antibacterial mechanism of penicillins occurs via the inhibition of the synthesis of the bacterial cell wall.
If bacteria had no cell wall, their cells would rupture due to the difference in osmolarity between the inside and the outside of the cell.
The substance that confers resistance and rigidity to the cell wall is the peptidoglycan.
This compound forms a mesh-like layer, consisting of glycosaminoglycan chains interlinked with short peptides.
The sugar component consists of alternating disaccharide of N-acetylglucosamine (NAG) and N-acetylmuramic acid (NAM), connected by a $\beta$(1,4) glycosidic bond.
There is a chain of four amino acids (both D- and L- ) attached to the N-acetylmuramic acid.

The structure is repetitive and resistant due to the third amino acid bond to the NAM of a NAM-NAG chain, which is tied to the fourth amino acid of a NAM on a parallel chain.
Penicillins block the formation of the cross-linking bonds within the peptidoglycan, therefore compromising its development in the cell wall (Figure 2.3).
The development of these bonds is catalysed by a class of enzymes called “penicillin binding proteins” (PBP).
These enzymes are able to remove one of the two residues of D-alanine placed at the end of the NAM pentapeptide.

\clearpage
\begin{figure}[htp]
\centering
\includegraphics[scale=0.50]{immagini gamalero/Peptidoglycan.jpg}
\caption{The peptidoglycan is synthesised in three different locations during three different stages: the first occurs in the cytoplasm, where the Lipid I is synthesised; the second takes place in the cytoplasmatic membrane where the Lipid I is transformed in the Lipid II and then flipped to the external side of the membrane, where the third stage occurs. During the last stage the Lipid II is linked to the nascent peptidoglycan by PBPs \cite{Pinho2013}}
\label{}
\end{figure}


$\beta$-lactam antibiotics are able to inhibit the PBP enzymes thanks to a competitive mechanism: since they are similar to the D-alanine D-alanine dimer, they are erroneously recognised by the enzyme as its substrates, causing the scission of the $\beta$-lactam antibiotic.
The splitted antibiotic bonds covalently with the enzyme, forming an acyl-enzyme stable complex: this blocked enzyme is no longer capable of catalysing the peptidoglycan synthesis reactions, causing the death of the growing bacterial cells \cite{kong2010beta}.

\subsection{Carbapenem antibiotics}
A new era of $\beta$-lactam antibiotics, the carbapenems, has begun after the discovery of Streptomyces cattleya and its antibiotic product, the thienamycin.
After thienamycin, numerous carbapenems were discovered (e.g. imipenem, meropenem) \cite{Birnbaum1985}.
Like the penicillins, carbapenems are part of the $\beta$-lactam class of antibiotics, with a broader spectrum of activity; moreover their effectiveness is less affected by many of the most common mechanisms of antibiotic resistance.

This class of antibiotics presents a remarkable activity against both Gram-positive organisms and Enterobacteriaceae, Pseudomonas aeruginosa, and Bacteroides \cite{Neu1985}.

Since carbapenems are more effective against infections caused by multidrug-resistant bacteria than other $\beta$-lactam antibiotics, they are primarily used in hospitalised patients.

\chapter{The antibiotic resistance and the $\beta$-lactamases}

Antibiotics are considered one of the major breakthroughs of modern medicine.
Their role has been essential in treating bacterial infections, and saving many lives.
Unfortunately, time has seen the emergence and spread of antibiotic resistances among bacteria, weakening their effects.
The development of combined resistances to multiple classes of antibiotics have brought to strains with multidrug-resistance (MDR) phenotypes, which can render traditional antibiotics completely ineffective \cite{Rossolini2014}.

Since $\beta$-lactams were the first antibiotics to be discovered, the resistance to this kind of antibiotics was the first to emerge, and to be understood.
The most effective way for bacteria to react to these antibiotics has been by producing $\beta$-lactamases.
These enzymes are able to hydrolyse the $\beta$-lactam ring of the antibiotics and, consequently, to inactivate them \cite{kong2010beta}.

Early works analysed and classified $\beta$-lactamases from a functional point of view.
Today, their classification is based on the amino acid homology between different antibiotics, and has resulted in four different major classes: “molecular classes A, C, and D include the $\beta$-lactamases with serine at their active site, whereas molecular class B $\beta$-lactamases are all metallo-enzymes with an active-site zinc” \cite{Queenan2007}.

\section{The carbapenemases}
Among these four classes, carbapenemases are part of the classes A, B, and D.

\subsection{Class A carbapenemases}
Bacteria expressing class A serine carbapenemases have low susceptibility to imipenem, and “their MIC (Minimum Inhibitory Concentration) can range from mildly elevated to fully resistant”.
This class is divided into three major families: NMC/IMI, SME and KPC enzymes.
All these carbapenemases are able to hydrolyse a broad variety of $\beta$-lactams (e.g. cephalosporins, penicillins, aztreonam) \cite{kong2010beta} \cite{Queenan2007}.

The first KPC-1 was discovered in a \emph{K. pneumoniae} isolated in North Carolina.
The KPC family can spread easily thanks to its location on plasmids \cite{Queenan2007}.

This family is able to hydrolyse penicillins, cephalosporins, monobactams, carbapenems, and even $\beta$-lactamase inhibitors.
Probably due to the few antibiotic options, there is a high mortality rate among patients infected with KPC positive bacteria \cite{MunozPrice2013}

Although 10 different variations of KPC have been described, the KPC-2 and KPC-3 are the most commonly identified \cite{WaltherRasmussen2007}.

\subsection {Class B: metallo-$\beta$-lactamases}
This class of beta-lactamases is resistant to the available beta-lactam antibiotics, but it is inhibited by metal ion chelators.
They have a broad spectrum of activity: most of them are able to hydrolyse carbapenems, cephalosporins and penicillins.
Chromosomal enzymes (including VIM, IMP, GIM, and SIM) found in environmental and opportunistic pathogenic bacteria, were the first metallo-beta-lactamases to be detected and studied.
These enzymes were usually expressed in conjunction with at least one serine beta-lactamase \cite{Queenan2007}.

\subsection{Class D: OXA beta-lactamases}
This group of serine Beta-lactamases was firstly identified in Enterocateriaceae and Pseudomonas aeruginosa, and constitutes a separate class.
The OXA beta-lactamases were described as penicillinases capable of hydrolysing oxacillin and cloxacillin.
Out of the 102 unique OXA sequences which have currently been identified, 9 are extended spectrum beta-lactamases and more than 37 are considered to be carbapenemases.
Among this group, the OXA-48 variant is plasmid encoded, and presents the highest hydrolysis rate among all the OXA enzymes.
The OXA-48 was firstly found in a clinical \emph{K. pneumoniae} isolated in Turkey \cite{Poirel2012}.

\chapter{Carbapenemases producing strains: Enterobacteriaceae}
The Enterobacteriaceae family includes a great variety of Gram-negative bacilli.
These bacilli can be aerobic or facultative anaerobic bacteria; most of them are mobile thanks to their flagella, but a few genera are nonmotile.
They are not-forming spore bacteria, and typically $0,5-1,5\mu m$ in size.
This family includes both harmless symbiotic bacteria and pathogenic bacteria (such as Salmonella, Escherichia coli, and Klebsiella).

\subsubsection{Pathogenicity of Enterobacteriaceae}

The pathogenicity of Enterobacteriaceae is strictly bound to the antigenic structure of their cell wall, which contains three main categories of antigens.
The K (capsular) antigen is a component of the polysaccharide capsule that surrounds the bacteria.
The main task of this capsular structure is to avoid phagocytosis and, consequently, the activation of the complement.
The 0 (somatic) antigen is responsible for the common symptoms of bacterial infections (i.e. fever, activation of the complement cascade, shock).
The outer membrane of the cell contains lipopolysaccharide (LPS),and the lipid A portion is endotoxic, while the O (somatic) antigen is serotype specific \cite{guentzel1996escherichia}.

The H antigens are flagellar proteins, and they are known for their invasiveness, and their ability to facilitate the ascension of uropathogenic bacteria from the bladder into the kidneys \cite{wiles2008origins}.

Some organisms have flagella distributed on their cell surface (e.g. Escherichia coli) and can present the H antigens, while others, which are nonmotile and nonflagellates (e.g. Klebsiella), have no H antigens.

One of the most common Enterobacteria that have been isolated in humans is Klebsiella.
Some strains belonging to this \emph{genus}, in particular the Klebsiella pneumoniae species, are showing resistance to almost every antibiotic available, and are responsible for the largest number of nosocomial infections.

\section{Klebsiella}
Klebsiella is a Gram-negative bacterium ascribed to the Enterobacteriaceae family.
This microorganism has two common habitats: the environment (e.g. surface water and soil), and the mucosal surfaces of mammals, such as humans.

Klebsiella mainly attacks immunocompromised individuals, especially if hospitalised.
It can be particularly troublesome when infecting premature infants (it is often involved in neonatal sepsis), and intensive care units \cite{podschun1998klebsiella}.
Among the Klebsiella species, Klebsiella pneumoniae is the most clinically relevant: it is responsible for 70$\%$ of human infections \cite{Pitout2015}.

\begin{figure}[htp]
\centering
\includegraphics[scale=0.30]{immagini gamalero/Klebsiella_pneumoniae.jpg}
\caption{Scanning electron microscope image of Klebsiella pneumoniae}
\label{}
\end{figure}

The most common sites of colonization in humans are the gastrointestinal tract, and the nasopharynx.
It can cause infections in the urinary tract, and also pneumonia \cite{Pitout2015, podschun1998klebsiella}.
In the last decades, controlling this kind of infection has been difficult due to the emergence and spread of drug-resistant strains of \emph{K. pneumoniae}.
The resistance of \emph{K. pneumoniae} to antibiotics (such as cephalosporins, fluoroquinoles, and trimethoprim-sulfamethoxazole) which are often used to treat this kind of infection, delays the start of a proper therapy.
This causes an increase in morbidity and mortality among patients.

Since carbapenems are often the last line of effective therapy available for the treatment of infections caused by multidrug-resistant (MDR) \emph{K. pneumoniae}, the emerging resistance to them is particularly concerning.
 \cite{Pitout2015}.

\subsection{Mechanisms of resistance to carbapenems} 

\emph{K. pneumoniae} resistances are due to different mechanisms.

The production of beta-lactamases, along with the occurrence of permeability defects, can lead to a reduced susceptibility to carbapenems.
These beta-lactamases enzymes can belong to the class A extende-spectrum beta-lactamases (ESBLs) or to the class C AmpC cephalosporinases.
There are also carbapenemases, belonging to the molecular classes A, B or D, that do not require additional permeability defects.

Among the beta-lactamases enzymes, there is the KPC-type, which was discovered for the first time in a sample of \emph{K. pneumoniae} isolated in North Carolina.
Researchers have currently described more than twenty different KPC variants, and the most common are KPC-2 and KPC-3.

%modificare
``\emph{K. pneumoniae} ST258 with KPC-2 and KPC-3 has contributed significantly to the dissemination of KPC enzymes worldwide.''
\cite{Pitout2015}.

The ST258 clone is the most common clone of KPC-producing \emph{K. pneumoniae}.

"Several different KPC-containing plasmids have been identified in ST258 and these plasmids often contain various genes encoding nonsusceptibility to different antimicrobial drugs" \cite{Pitout2015}.

Another important group of beta-lactamases are the metallo-beta-lactamases (MBLs).
Bacteria with MBLs are often resistant to penicillins, carbapenems, cephalosporins, and cephamycins but remain susceptible to monobactams. Moreover, they are inhibited by metal chelators (e.g. EDTA and dipicolinic acid).

OXA-48 beta-lactamase is the only class D carbapene-hydrolysing beta-lactamase isolated from \emph{K. pneumoniae}.
This enzyme hydrolyses beta-lactams such as penicillins, hydrolyses carbapenems, and cephalosporins.


\subsection{KPC epidemiology}

The first KPC-producing strain was isolated in 1996 in a hospital in North Carolina, and was followed by a report describing KPC-positive isolates from New York City hospitals.
KPC-positive bacteria can be isolated in urine, respiratory, blood, and wound samples.
In Italy, the first KPC-positive \emph{K. pneumoniae} was isolated in 2008 in Florence.
The isolate presented a KPC-3 enzyme, with the corresponding gene located in transposon Tn4401.
In 2009, a second report showed two KPC-2 positive \emph{K. pneumoniae} isolated in Rome.
Between 2009 and 2011, thanks to an active surveillance in two hospital in Padua, almost two hundred cases were identified.
\cite{MunozPrice2013}
The last ten years have shown a worldwide increasing spread of CPE strains.
This phenomenon has had particular relevance in countries such as the United States of America, Israel, Puerto Rico, Colombia, and Greece.
The spread of carbapenemases among different strains is probably due to the dissemination of mobile genetic elements that can transfer their resistance genes to other microorganisms.
One of the causes for the spread of KPC epidemic clones is patients being transferred between different hospitals, or different countries \cite{circolare2013}.

\subsection{Treatment of infections due to carbapenemases-producing \emph{K. pneumoniae}}
Mortality rates due to \emph{K. pneumoniae} infections are usually between 23 and 75$\%$ \cite{karaiskos2014multidrug}.
None of the strategies using the currently available antibiotics is optimal to cure carbapenemase-producing K.pneumoniae infections, and single antibiotic therapies are usually ineffective.
Severe infections due to a carbapenemase-producing K.pneumoniae strain can justify the use of a combination therapy that includes colistin and a carbapenem, or an aminoglycoside.
Often, the only antibiotics that show in vitro activity are polymyxins (e.g., colistin or polymyxin B), tigecycline, fosfomycin, and sometimes selected aminoglycosides \cite{rodriguez2015diagnosis}.
Since carbapenemase-producing bacteria are often resistant to other antibiotic classes too (e.g fluoroquinolones and aminoglycosides), it is important to make susceptibility tests for antibiotics such as polymyxins (e.g. colistin), fosfomycin, tigecycline, and rifampin.
These antibiotics can represent the last resort for treating such infections \cite{adams2009activity}.
Even if KPC-producing strains are usually resistant to all beta-lactam antibiotics, some of them still show a certain susceptibility to temocillin.
Moreover, NDM, VIM, and IMP producers are susceptible to aztreonam, while OXA-48 producers should be tested to verify if they are susceptible to the expanded-spectrum cephalosporins \cite{girlich2009ctx}.
Since single antibiotic therapies have proved not to be very effective, combined therapies are often prescribed, as they can maximise bacterial killing (synergistic effect), while reducing bacterial resistances \cite{Pitout2015}.
The mortality rate appears to be significantly lower in patients that have undergone combination therapies \cite{tzouvelekis2014treating, zavascki2013combination}, and the best antibiotic associations result from the administration of two molecules showing in vitro activities against carbapenemase-producing strains \cite{falagas2013antibiotic, tzouvelekis2014treating}.
The colistin (polymyxin E) antibiotic has been discovered more than 60 years ago \cite{karaiskos2014multidrug, rodriguez2015diagnosis}.
This moleceule is often used in combination therapies, and is significantly effective against various carbapenemase-producing strains \cite{falagas2013antibiotic, temkin2014carbapenem}.
Even if it shows nephrotoxicity as a side effect, and a poor lung penetration, colistin is the most popular antibiotic for treating carbapenemase-producing \emph{K. pneumoniae} infections \cite{karaiskos2014multidrug, rodriguez2015diagnosis}.
Unfortunately, due to the increased use of this antibiotic, 
colistin-resistant K pneumoniae strains have already been reported \cite{mammina2012ongoing}.
Another antibiotic available in Europe is fosfomycin, commonly used in combination with tigecycline and colistin, in order to treat MDR bacteria infections \cite{pontikis2014outcomes}.
Gentamicin is still effective against some KPC and OXA-48
producers.
Despite the ability to produce carbapenemases, carbapenems can be used as an antibiotic option, when the MICs of carbapenems are $\le 8mg/l$, against carbapenemase-producing \emph{K. pneumoniae}.
This therapy can be successful when combined with a second antibiotic, or when a prolonged intravenous infusion regimen is used \cite{tzouvelekis2014treating, daikos2014carbapenemase, tumbarello2012predictors}.
As studies lead with an animal model of infection (i.e. mouse pneumonia) have shown, a dual-carbapenem therapy (i.e. meropenem and ertapenem) could be effective \cite{wiskirchen2014vivo}.
Meropenem has been shown to retain its efficacy, whereas most probably ertapenem could act ad a ``suicide'' molecule for carbapenemase activity.

The efficacy of this therapy, which involves double-carbapenem, has been proved in human patients infected with KPC-producing strains \cite{giamarellou2013effectiveness}.
Other useful beta-lactams are extended-spectrum cephalosporins, effective against OXA-48 producers without ESBLs \cite{mimoz2012broad}, and aztreonam, which can be an option for treating MBL producers infections \cite{nordmann2011emerging}.

\chapter{Materials and methods}

The emergence of carbapenem-resistances in enterobacteria strains represents a significant clinical problem, since carbapenem antibiotics are the reference drugs for treating infections caused by multiresistant enterobacteria.

The carbapenem resistant enterobacteriaceae (CRE), especially if carbapenemases producers, are a danger for public health for different reasons:
\begin{itemize}
\item infections due to enterobacteria strains are frequent in both hospital and community settings, and the spread of CRE infections makes treating patients difficult;
\item the mortality rate attributable to CRE infections usually is $20-30\%$  \cite{carmeli2010controlling}, but it can reach up to $70\%$ during bacteremia \cite{mouloudi2010bloodstream};
\item these microorganisms can easily spread across different patients, and their resistances can be transmitted to other microorganisms through plasmids.
\end{itemize}

Experiences in different hospitals and countries have shown that it is possible to reduce, or even eradicate the spread of these microorganisms.
This can be achieved thanks to aggressive measures of control in the sanitary environment, aimed at promptly detecting the presence of the infection and its hosts.
After identifying the infection, it is necessary to adopt measures to contain its spread (i.e. isolation, hand hygiene, environmental decontamination) \cite{gupta2011carbapenem}.

Enterobacteriaceae can produce different carbapenemases, especially belonging to classes A and B.
More rarely, the carbapenem resistance can be due to resistance mechanisms to beta-lactam antibiotics combined with a porin deficit, or to D class carbapenemases (e.g. OXA-48).

It is important to monitor the Enterobacteriaceae isolates with the lowest level of resistance.
Suspicion that the isolates are carbapenemase-producing Enterobacteriaceae should arise when the MIC of the carbapenem is higher than epidemiological cut-off (ECOFF) of the respective wild-type strains.
The ECOFF values define the top end of the wild-type distribution: microorganisms with MIC values higher that their ECOFF have most likely acquired some type of resistance.
In order to find more effective therapies, and avoid the spread of antibiotic resistances, screening tests are required.
Different phenotypic and genotypic analyses are used in order to identify the pathogens and their antibiotic resistances.

For this project, various techniques of on plate cultivation, and automated tools (i.e. Vitek2, and the GeneXpert system) have been used.

\section{Sowing on plate}
Initially the material is s on a culture medium containing a concentration of nutrients suitable for bacterial growth.
Depending on the biological material, suitable solid media can be used. 
Specific bacteria can be grown on media containing specific substances:
\begin{itemize}
\item blood samples from central vessels and peripheral veins are sown on chocolate agar, McConkey agar, Columbia agar with 5$\%$ Sheep Blood (COL-S), and Sabourad;
\item bronchoalveolar lavage and sputum samples need to be diluted with Sputasol (1:1) and, if the sample is considered suitable, are sown on COL-S, Columbia CNA agar, McConkey agar, and Chocolate agar with Bacitracin;
\item oropharyngeal swab samples are sown on TSA-II and Sabourad;
\item nasal swab samples are sown on MRA agar;
\item rectal swab samples are sown on ESBL agar and McConkey agar with a disk of Meropenem antibiotic;
\item cavitary liquid samples (i.e. pleural, pericardial, peritoneal, and synovial) are sown on COL-S, Schaedler KV Agar with 5$\%$ Sheep Blood, and bottles for both anaerobic and aerobic growth.
\end{itemize}

In order to identify colonies of \emph{K. pneumoniae}, the most suitable media are:
\begin{itemize}
\item McConkey agar.
This is a selective and differential culture terrain for the identification of Gram-negative and enteric bacteria.
Since it contains crystal violet and bile salts, it is able to inhibit the growth of Gram-positive bacteria. 

This medium contains lactose as the only source of carbohydrates, and the neutral-red pH indicator.
The lactose fermenting bacteria, such as \emph{K. pneumoniae}, appear as red colonies (Figure 5.1) without precipitation of bile salts;

\begin{figure}[htp]
\centering
\includegraphics[scale=0.250]{immagini gamalero/KlebsiellaMAC.jpg}
\caption{Klebsiella pneumoniae growing on MacConkey Agar}
\label{}
\end{figure}

\clearpage
\item ESBL agar.
This is a chromogenic medium, specifically designed for the extended-spectrum beta-lactamases producing Enterobacteria. This terrain allows a direct identification, already after 18-24 hours of incubation.
If sown on this terrain, the \emph{K. pneumoniae} colonies show a green/blue colour, while the E. coli colonies show a pink/burgundy colour.
\end{itemize}

\section{Screening tests in infected patients}
The carbapenemase-producing strains can spread rapidly in health facilities, and have a high clinical and epidemiological impact.
The study of infected patients is crucial because they are the main source of contamination.
Screening tests should be performed by examining cultures grown on rectal swab samples, even if other locations can also be colonized (oropharynx, bladder, etc.).

Screening tests need to be sensitive, to give a rapid response, and have a low cost.
Meropenem is suggested as the reference molecule to detect the presence of carbapenemases. This molecule offers the best compromise between sensitivity and specificity. 
It is advisable to suspect the production of carbapenemases when there is a low sensitivity to meropenem with MIC $\ge 0,5 mg/l$, or a diameter of the inhibition zone $\le 25 mm$ (EUCAST).
These values are particularly significant for E. coli and \emph{K. pneumoniae}, while they have to be assessed with caution for other species. Sometimes meropenem MIC values of $0,5-1 mg/l$ can be due to an over-expression of the AmpC beta-lactamase, associated with a loss of porines.

Furthermore, an inhibition zone of $\le 25 mm$ may not highlight some strains, which have a diameter of the inhibition zone of $24-26 mm$. Therefore, it is suggested to use a screening threshold with a wider diameter ($\le 27 mm$).
Meropenem antibiotics have a better specificity than imipenem and ertapenem.
The use of ertapenem as an indicator with the same MIC value determines an increased sensitivity, but also a decreased specificity.

The screening tests are made by:
\begin{itemize}
\item seeding the bacterial strains on chromogenic media;
\item seeding on McConkecy agar the bacterial strains with a disk of meropenem antibiotic;
\item enrichment in liquid media added with a carbapenem and seeding the bacterial strains on McConkey agar.
\end{itemize}

\subsection{Seeding on chromogenic media}
This method involves the direct seeding of the swab on specific chromogenic media with low sensitivity to carbapenems.
This type of growth-media allows to easily recognise suspect colonies, as well as the species involved.
The sensitivity and specificity of the media need to be carefully evaluated:  using media with lower specificity can delay the disclosure of results.

\subsection{Direct seeding on McConkey agar}
The method involves streaking the swab on a McConkey agar plate.
During the seeding half of the plate is covered  with the swab (usually a rectal swab), rotated of 90° and then drawn with a sterile loop. 
This allows to spread the bacteria all over the plate, and to get single pure colonies.
After the seeding, a $10\mu g$ Meropenem disk is positioned in the more densely sowed area.
The plate is incubated for $16-24$ hours at $\sim 35°C$..

During this work colonies with the typical Enterobacteriaceae morphology have been considered, grown inside the diameter of the inhibition zone.
This assay allows to simply recognise the suspect colonies, and to obtain results the results in a short time. 

\subsection{Enrichment in liquid media added with a carbapenem and seeding on McConkey agar}

This method, suggested by the Centers for Disease Control and Prevention, involves seeding the rectal swab in $5ml$ of Trytic Soy Broth, containing a $10\mu g$ meropenem disk.
This is followed by 18 hours of incubation at $35°C$.
Subsequently, a subculture is seeded on a McConkey agar plate, then incubated at room temperature for $24-48$ hours. (citare)
This test has a good sensitivity and helps to recognise the colonies, but it requires a higher operational level than the previous methods.

\section{Determination of antibiotic resistance/sensitivity profile}
This test allows the in vitro evaluation of the bacteria sensitivity profile to various antibiotics.
During this test the bacterial strains is exposed to standard concentrations of different antibiotics.
The most common methods, used in the microbiology laboratories are based on diffusion (i.e. the Kirby Bauer method) (Figure 5.2), or dilution (this method can be automated).

\begin{figure}[htp]
\centering
\includegraphics[scale=0.800]{immagini gamalero/Kirby_Bouer.jpg}
\caption{Kirby Bouer Test}
\label{}
\end{figure}

The first method is based on the contact between the antibiotic disk and the bacterial colonies, followed by the measurement of the diameter of their growth inhibition halo. 
The second method allows to evaluate the Minimal Inhibitory Concentration for specific antibiotics (i.e. the lowest concentration of antibiotic necessary to inhibit the bacterial growth).
The diameter of the inhibition zone, as well as the MICs, are related to standard threshold values (i.e. breakpoints) for different microorganism-antibiotic combinations.

The breakpoint results can be converted into three categories of interpretation:
\begin{itemize}
\item sensitive (S), when the bacteria are killed or inhibited by the antibiotic;
\item intermediate (I), when the bacteria need high concentrations of antibiotic to be inhibited;
\item resistant (R), when the bacteria grow in presence of the antibiotic.
\end{itemize}

\subsection{Synergy tests}
The synergy test is a combined method that tests the strains' resistance to carbapenemase, when exposed to a specific inhibitor for the carbapenemases:
\begin{itemize}
\item the ethylenediaminetetraacetic acid (EDTA) and dipicolinic acid (DPA) are able to inhibit metallo beta-lactamase producing bacteria;
\item the phenylboronic acid (APB) is able to inhibit KPC enzymes, but it can lead to false positive results in presence of AmpC;
\item cloxacillin (CLX), when combined with phenylboronic acid, highlights AmpC producing strains associated with porines loss;
\item temocillin highlights the high resistance level of OXA-48 strains.
\end{itemize}

During the test procedure some pure colonies are taken with a sterile loop, and diluted in $3ml$ of sterile saline until a $0.5$ McFarland concentration.
Then, a sample is taken from the solution containing the inoculum, and is sown on an Agar Muller Hinton (MHA) plate (this medium contains meat extract, casein acid hydrolysate, and soluble starch). 

Here, at a distance of $15-22 mm$, the following antibiotic disks are placed:
\begin{itemize}
\item meropenem (MRP10);
\item meropenem and boronic acid (MR+BO);
\item meropenem and dipicolinic acid (MR+DP);
\item meropenem and cloxacillin (MR+CL);
\item meropenem and ethylenediaminetetraacetic acid (MR+ED)
\end{itemize}

The plate is then incubated for $18-24$ hours in aerobic environment at $35°C$.
After the incubation, the inhibition zones are measured: these are proportional to the sensitivity of the microorganism to the specific antibiotic. 
Similarly, the inhibition zone is proportional to the minimum inhibitory concentration (Figure 5.2).

\begin{figure}[htp]
\centering
\includegraphics[scale=0.93]{immagini gamalero/5disks.jpg}
\caption{Determination of antibiotic resistance/sensitivity profile}
\label{}
\end{figure}

\clearpage
\subsection{Automated systems}
\subsubsection{VITEK® 2: recognition of phenotypic variants}

\begin{figure}[htp]
\centering
\includegraphics[scale=0.500]{immagini gamalero/Vitek_III.jpg}
\caption{The Vitek2}
\label{}
\end{figure}

VITEK® 2 is a fully automated system that performs bacterial identification, and antibiotic susceptibility testing \cite{Vitek2}.
The procedure involves the use of VITEK® 2 GP or GN cards for the microbial identification (ID), and VITEK® 2 AST cards, specific for the antibiogram.
Every card shows a unique bar code that act as a link between the card and the patient number.

\begin{figure}[htp]
\centering
\includegraphics[scale=0.20]{immagini gamalero/Vitek_Cards.jpg}
\caption{Cards for identification of Gram-negative bacteria and the determination of their antibiotic resistance/sensitivity profile.}
\label{}
\end{figure}

These cards have 64 wells containing biochemical reagents or antibiotics. Every card has also a small tube that transfers the suspension into the wells.
The suspension should have a concentration of $0,5-0,6$ McFarland, and the colonies diluted in the saline buffer must be pure.

Once the cassette is loaded, the instrument incubates it and reads each card.
By analysing the fluorescent substrate which develops onto both cards (GP or GN, and AST), the instrument is able to identify the microorganism and calculate its MIC, converted into one of the three categories of interpretation (S, I, or R).

\subsubsection{GeneXpert®: recognition of genotypic variants}
The Cepheid Xpert® CARBA-R test is a diagnostics qualitative analysis tool, designed to detect and differentiate gene sequences that give resistance to carbapenem antibiotics.
This test can identify the following genes: blaKPC, blaNDM, blaVIM, blaOXA-48, and blaIMP-1.

\begin{figure}[htp]
\centering
\includegraphics[scale=1.00]{immagini gamalero/genexpert.jpg}
\caption{The Gene Xpert}
\label{}
\end{figure}

The Xpert® CARBA-R test performs a PCR (polymerase chain reaction) multiplex reaction of amplification, to detect specific DNA target sequences.
Every instrument contains a syringe for dispensing the sample, a ultrasonic horn for the cell lysis, and a thermal cycler for executing the PCR and detecting the target.
While preparing the sample, some pure colonies are diluted in saline solution to a concentration of $0,5$ McFarland. 
$20\mu l$ of the resulting solution are added to a specific reagent, then shaken for 10 seconds with a vortex mixer, and finally put in a specific disposable cartridge.

The GeneXpert platform performs the subsequent steps: a filter captures the bacterial cells, and breaks them with ceramic marbles and a ultrasonic horn.

The solution containing the DNA is combined with the PCR dried reagents, and transferred into an optic tube were the real time PCR is performed.
The whole process requires 60 minutes.

%io direi "dati ottenuti e conclusioni" la divisione in due capitoli non mi piace tanto perchè non ho veramente molto da mettere nelle conclusioni e secondo me sta male fare un capitolo intero per scrivere 4 frasi.
\chapter{lavoro svolto e dati ottenuti}

The work was carried out in the microbiology laboratory at the hospital ``SS. Antonio e Biagio e C. Arrigo'' in Alessandria, between July 2016 and February 2017.

During this period of time (??) 38 samples have been taken into account starting from a larger number of samples.
Every sample from a patient with a suspected Enterobacteriaceae infection (?) was submitted to the synergy test.
Only the samples that showed a synergistic effect between meropenem and phenylboronic acid (??), that indicates a possible presence of KPC enzymes, or no synergistic effect, that indicates a possible presence of OXA-48 enzymes, were taken into account.

The goal was to highlight which carbapenemases was the cause of the most diffused antibiotic resistances found in \emph{K. pneumoniae} in the local territory.

The 38 samples were analysed, both from a phenotypic and genotypic point of view, and were taken from both male (24) and female (14) patients.

Most of the patients were hospitalised, and especially from wards with long term stay, or had undergone surgical operations.
The most represented wards were surgery (12), intensive care (7), and urology (7).

%immagine (Sample_wards_of_origin)

The samples were mainly rectal swabs (16), blood samples (10), and urine samples (7).

%immagine (Samples)
%manca blood samples

After the on plate isolation of pure colonies, phenotypic tests were performed in order to identify the antibiotic resistances. 
All samples analysed were resistant to Piperacillin/Tazobactam and Cefotaxime, while antibiotics that still show a certain degree of effectiveness are Colistin, Fosfomycin, and Amikacin.

Among the 38 samples, 8 showed resistances to all the antibiotics tested.

%aggiustare grafico (Vitek2)

After the phenotypic identification, the colonies were subjected to genotypic analysis to identify which antibiotic-resistant molecules were produced by the bacteria.

Among the 38 samples, 34 were KPC positive, while 2 samples were OXA-48 positive.
No sample was found to produce VIM, IMP or NDM carbapenemases.

%immagine Results_GeneXpert

\chapter{Fairy tales and morals}
The results obtained from the 38 samples fit correctly into the framework (??) outlined so far thanks to the literature. (??)

The data collected show how antibiotic resistances in \emph{K. pneumoniae} are extremely widespread (?), and confirm that these resistances are due primarily to KPC enzymes.

Since the growing spread of these antibiotic resistances is influenced by the selective pressure of broad-spectrum antibiotic therapies, this work also wanted to highlight how such therapies have become inadequate to control the spread of \emph{K. pneumoniae}.

The awareness of the spread of this bacteria in a hospital setting, and its resulting clinical implications, makes the role of the microbiology laboratory fundamental. 

An early detection of these infections makes it possible to implement programs for their containment and control, in addition it allows the start of targeted therapies.






























\bibliography{references}
\bibliographystyle{unsrt}

\end{document}